### ISSUE TITLE
VS Code Extension Usage needs to be more prominent

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-che/che#15760]
VS Code extensions are our most important way to extend Che. However, I was not able to find anything about this in the Che 7 doc. The information is there: I found a direct link on the Che repository (https://www.eclipse.org/che/docs/che-7/using-a-visual-studio-code-extension-in-che/).

The problem is that in the sidebar navigation on (https://www.eclipse.org/che/docs/che-7) there is no mention of VS Code extensions in neither the contributor guide nor the administrator guide.

**Comment by benoitf** [comment]
I think it's already covered by https://github.com/eclipse/che/issues/14119

**Comment by l0rd** [comment]
@tsmaeder it's in the end user guide (because it's about using vscode extensions vs authoring them)

![image](https://user-images.githubusercontent.com/606959/72843992-f074b800-3c9b-11ea-991b-26a60afd7490.png)

But yes I think we should have something in the contributor guide as well and as @benoitf mentioned that 's basically what's asked in #14119 

cc @themr0c the vscode extensions are currently the main way (if not the only one) to extend che workspaces. Talking about che-theia plugins doesn't make sense. It's confusing. IMHO we should address that as soon as possible.

**Comment by l0rd** [comment]
@themr0c @tsmaeder leaving this open for now but we should close it if it doesn't provide anything that's not in #14119 already.

**Comment by ericwill** [comment]
Forgive me if this seems like a silly question, but what exactly is supposed to be added to the docs for #14119? How to write VS Code extensions (something like [this](https://code.visualstudio.com/api/get-started/your-first-extension)), how to write a VS Code extension inside Che, or something else?

Maybe the existing portion about VS Code extensions in Che should be expanded upon (how to build sidecar containers, where to add the code for them, etc.) and moved into the Contributor Guide section?

**Comment by ericwill** [comment]
@l0rd is this one being handled as part of the tiger team?

**Comment by l0rd** [comment]
> @l0rd is this one being handled as part of the tiger team?

No. It's not labelled `area/doc` and has no priority. It's out of the radar. 

Those are some vs code extension issues currently in the backlog:
- https://github.com/eclipse/che/issues/17377
- https://issues.redhat.com/browse/RHDEVDOCS-1924
- https://github.com/eclipse/che/issues/15982
- https://issues.redhat.com/browse/RHDEVDOCS-1637
- https://issues.redhat.com/browse/RHDEVDOCS-1901

**Comment by themr0c** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-1579
